test/ExpressionExplorer.jl

```diff
@@ -115,6 +115,7 @@ Some of these @test_broken lines are commented out to prevent printing to the te
         @test testee(:((x, y), a, (b, c) = z, e, (f, g)), [:z, :e, :f, :g], [:x, :y, :a, :b, :c], [], [])
         @test testee(:((x[i], y.r), a, (b, c) = z, e, (f, g)), [:x, :i, :y, :z, :e, :f, :g], [:a, :b, :c], [], [])
         @test testee(:((a[i], b.r) = (c.d, 2)), [:a, :b, :i, :c], [], [], [])
+        VERSION > v"1.7.0-0" && @test testee(:((; a, b) = x), [:x], [:a, :b], [], [])
```

Comment:
not sure if the version check is needed.

I'd try what happens on CI, but GHA workflows are still disabled for me.